### DISCUSSION TITLE
I want to be able to translate the web panel

### DISCUSSION THREAD

**Discussion by flavioxx** [immense/Remotely#795] [Q&A]
I want to be able to translate the web panel
I noticed that remotely has folders with country acronyms, I think they are the translations, how do I use them? I wanted to use pt-BR, or does anyone have another alternative?
Also, if anyone can help me with this, I'm willing to pay for the help, my contact telegram @flmh1995 or leave your contact and I'll also contact you by email or whatever.

**Comment by alencarwell** [comment]
Hello, i need this too...
